js/view/signup.js

```diff
@@ -0,0 +1,90 @@
+const SignUp = {
+  render: () => {
+    return `
+       <main>
+        <section class="section">
+        <form class="form">
+          <div class="form__title">
+            <span class="form__title-text">Sign Up</span>
+          </div>
+          <div>
+            <div class="form__input__wrapper">
+              <input
+                class="form__input"
+                type="text"
+                placeholder="First Name"
+                id="name"
+                autocomplete="on"
+                required
+              />
+              <p class="error-message__name"></p>
+            </div>
+            <div class="form__input__wrapper">
+              <input
+                class="form__input"
+                type="text"
+                placeholder="Last Name"
+                id="surname"
+                autocomplete="on"
+                required
+              />
+              <p class="error-message__surname"></p>
+            </div>
+            <div class="form__input__wrapper">
+              <input
+                class="form__input"
+                type="email"
+                placeholder="Email"
+                id="email"
+                autocomplete="on"
+                required
+              />
+              <p class="error-message__email"></p>
+            </div>
+            <div class="form__input__wrapper">
+              <input
+                class="form__input"
+                type="password"
+                placeholder="Password"
+                title="Password must contain minimum eight characters, at least one letter and one number"
+                id="password"
+                autocomplete="on" 
+                required
+              />
+              <div class="input__svg input__svg_close"></div>
+              <p class="error-message__password"></p>
+            </div>
+            <div class="form__input__wrapper">
+              <input
+                class="form__input"
+                type="password"
+                placeholder="Confirm Password"
+                id="confirmPass"
+                autocomplete 
+                required
+              />
+              <div class="input__svg input__svg_close"></div>
+              <p class="error-message__confirmPass"></p>
+              
+            </div>
+          </div>
+
+          <button class="form__button form__button_register" type="submit" data-name="/doctor">
+            Sign up
+            <svg class="button__svg">
+              <use href="./svg/sprite.svg#icon-angle-right-b"></use>
+            </svg>
```

Comment:
Use :before on buttons, cause this arrow doesnt have any interactions on it, so it could be moved to CSS